packages/pipeline/src/Counter.tsx

```diff
@@ -101,29 +34,38 @@ const getStatefulStyles = (state: StateForStyles) => ({
  */
 const Counter = forwardRef(
   (
-    { className = '', children, size, ...rest }: CounterProps,
+    { className = '', children, ...rest }: CounterProps,
     ref: Ref<HTMLDivElement>,
   ): ReactElement => {
-    const { base: baseStyle, chevron: chevronStyle } = getStatefulStyles({
-      size,
-    });
+    const { theme, size } = useContext(PipelineContext);
+    const Icon = segments[size];
 
     return (
       <div
         {...rest}
-        {...counterAttr.prop}
         data-testid="pipeline-counter"
-        className={cx(baseStyle, className)}
+        className={cx(
+          counterBaseStyles,
+          counterSizeStyles[size],
+          counterThemeStyles[theme],
+          className,
+        )}
         ref={ref}
+        // Adding this so on focus the tooltip will show up
+        // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
+        tabIndex={0}
       >
+        {/* The counter is a unique shape with focus and hover state that are not easily accomplished with css alone so we are using an SVG. The reason we have 2 SVGs is to mimic the behavior of a div expanding horizontally when the text increases since an SVG does not stretch horizontally while maintaining its height.*/}
+        <div className={counterSvgBaseStyles}>
+          <div className={counterSvgColStyles}>
+            <Icon className={cx(svgLayer1Styles)} />
+          </div>
+          <div className={counterSvgColStyles}>
+            <Icon className={cx(svgLayer2Styles)} />
```

Comment:
I assume the `cx` was relevant at some point, but it looks like we can remove them now?